src/main/java/seedu/planitarium/family/Family.java

```diff
@@ -94,6 +95,16 @@ public void addPerson(int group, String name, Boolean isSilent) {
             return;
         }
         String generation = getGenerationName(group);
+        printPersonAdded(name, generation);
+    }
+
+    /**
+     * Prints the name of the person added, and to which array list.
+     *
+     * @param name The name of the person added
+     * @param generation The group the person is added to
+     */
+    private void printPersonAdded(String name, String generation) {
         System.out.println(name + " has been successfully added to " + generation);
     }
```

Comment:
Possibly consider using String.format instead and shifts the value into constants.

E.g. String.format(CONSTANT, args)